operators/src/engine/execution_context.rs

```diff
@@ -105,9 +106,12 @@ impl MockExecutionContext {
     }
 
     pub fn mock_query_context(&self, chunk_byte_size: ChunkByteSize) -> MockQueryContext {
+        let (trigger, valve) = Valve::new();
         MockQueryContext {
             chunk_byte_size,
             thread_pool: self.thread_pool.clone(),
+            valve,
+            valve_trigger: Some(trigger),
```

Comment:
strange name, can you name it more accordingly?
stream cancel callback or idk

Reply:
we can talk about the terminology if we decide we actually want to do it this way

Reply:
I changed the name to `abort_registration` and `abort_trigger` db426f1844ce4883ad3021252bbf6bbfee69eeb5